server/game/cards/14.4-CoC/SturdyTetsubo.js

```diff
@@ -0,0 +1,21 @@
+import { AbilityTypes } from '../../Constants.js';
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class SturdyTetsubo extends DrawCard {
+    setupCardAbilities() {
+        this.whileAttached({
+            effect: AbilityDsl.effects.gainAbility(AbilityTypes.Reaction, {
+                title: 'Make opponent discard 1 card',
+                when: {
+                    afterConflict: (event, context) => context.source.isParticipating() && event.conflict.winner === context.player
```

Comment:
This should be checking whether the character won, not the player. You could equip this to an opponent's character in theory. It's always better to literally copy the card text, makes for less mistakes.